README.md

```diff
@@ -0,0 +1,2 @@
+# *imhungry* Android app
+Let *[imhungry](http://www.imhungry.io/)* take the guess work out of deciding what to eat!
```

Comment:
You should use `https` in the link, and drop the `www`.

Reply:
`https://imhungry.io/` is a dead link.

Reply:
They all are right now, but `https://imhungry.io` will be the one we end up using. The current "live" one is just what namecheap threw on it by default.